articles/service-fabric/how-to-managed-cluster-availability-zones.md

```diff
@@ -218,6 +218,26 @@ Requirements:
    ```
    If you run in to any problems reach out to support for assistance.
 
+## Enable FastZonalUpdate on Service Fabric managed clusters (preview)
+Service Fabric managed clusters support faster cluster and application upgrades by reducing the max upgrade domains per availability zone from five to three.  
+
+The update should be done via ARM template by setting the fastZonalUpdate property to “fast”.  The Service Fabric managed cluster resource apiVersion should be 2023-01-01-preview or later
```

Comment:
the property name is incorrect/doesn't match with the template

Reply:
updated